doc/source/ray-core/examples/batch_prediction.ipynb

```diff
@@ -0,0 +1,283 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Batch Prediction\n",
+    "\n",
+    "```{tip}\n",
+    "You may want to develop batch prediction with [Ray Datasets] (https://docs.ray.io/en/latest/data/examples/nyc_taxi_basic_processing.html#parallel-batch-inference) instead, which will enable you to build it more easily and faster. Use Ray Core APIs directly only when that’s not an option for your use case. \n",
```

Comment:
If they cannot use dataset directly due to missing data source, should we suggest them to add their custom data source (we should have a doc for this as well) instead of using Ray Core?